Products/ZenModel/TrendlineThreshold.py

```diff
@@ -0,0 +1,90 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2015, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+
+__doc__= """TrendlineThreshold
+Makes a threshold comparison against a projected trendline
+"""
+
+from Globals import InitializeClass
+from MinMaxThreshold import MinMaxThreshold, MinMaxThresholdInstance
+from ThresholdInstance import ThresholdContext
+from Products.ZenEvents.ZenEventClasses import Capacity
+
+import logging
+log = logging.getLogger('zen.TrendlineThreshold')
+
+PROJECTION_TIME_UNITS = ['days', 'weeks', 'months']
+PROJECTION_ALGORITHMS = ['linear']
+
+class TrendlineThreshold(MinMaxThreshold):
```

Comment:
Eventually this will make a datasource that stores the projection variables (e.g. "m" and "b" for mx+b) but for right now just signifies to the visualization library to render a projection.